app/src/main/java/ar/com/wolox/android/example/ui/login/ILoginView.java

```diff
@@ -9,11 +9,15 @@
 
     void onEmptyPassword();
 
+    void onEmptyUsernameAndPassword();
+
     void onWrongUsernameFormat();
 
+    void goToTermsConditionsScreen();
+
+    void onUsernameAlreadyStored(String usernameStored);
```

Comment:
User nullabilities here to determine if `usernameStored` should be Nonnull or nullable